### PR TITLE
Fix bug 1567400 - Provide the VCS identity via the environment variab…

### PR DESCRIPTION
**Pull request by jotes**
@mathjazz r?

### CODE REVIEW

File: pontoon/sync/vcs/repositories.py [hunk 1]
```diff
@@ -154,9 +154,9 @@ def commit(self, path=None, message=None, user=None, branch=None):
         git_cmd = [
             "git",
             "-c",
-            "user.name=Mozilla Pontoon",
+            f"user.name={settings.SYNC_VCS_NAME}",
```
Comment (mathjazz) [inline pontoon/sync/vcs/repositories.py:157]: Should this actually be `user.first_name`?

File: pontoon/sync/vcs/repositories.py [hunk 2]
```diff
@@ -154,9 +154,9 @@ def commit(self, path=None, message=None, user=None, branch=None):
...
             "-c",
-            "user.email=pontoon@mozilla.com",
+            f"user.email={settings.SYNC_VCS_EMAIL}",
```
Comment (mathjazz) [inline pontoon/sync/vcs/repositories.py:159]: Same as above: Should this actually be `user.email`?

File: pontoon/settings/base.py [hunk 3]
```diff
@@ -47,6 +47,9 @@ def path(*args):
 # A list of project manager email addresses to send project requests to
 PROJECT_MANAGERS = os.environ.get("PROJECT_MANAGERS", "").split(",")
 
+# Email from which new locale requests are sent.
+LOCALE_REQUEST_EMAIL = os.environ.get("LOCALE_REQUEST_EMAIL", "pontoon@example.com")
```
Comment (mathjazz) [inline pontoon/settings/base.py:51]: Maybe `LOCALE_REQUEST_FROM_EMAIL`, to make it more verbose?

File: pontoon/settings/base.py [hunk 4]
```diff
@@ -736,6 +739,9 @@ def _allowed_hosts():
 
 SYNC_LOG_RETENTION = 90  # days
 
+SYNC_VCS_NAME = os.environ.get("SYNC_VCS_NAME", "Pontoon")
+SYNC_VCS_EMAIL = os.environ.get("SYNC_VCS_EMAIL", "pontoon@example.com")
```
Comment (mathjazz) [inline pontoon/settings/base.py:743]: I'd add call these `VCS_SYNC_X` and move them up, right under `LOCALE_REQUEST_EMAIL` (to keep account/email settings together).

I'd also add a comment on top, e.g.:
```
VCS identity to be used when committing translations.
```

**Comment by jotes** [comment]
@mathjazz I've applied the requested changes, can you take a look again?

**Comment by mathjazz** [comment]
Nice work!

As mentioned above, we should document these new variables:
https://github.com/mozilla/pontoon/pull/1649#pullrequestreview-435251400

**Comment by jotes** [comment]
@mathjazz Could check descriptions of the new settings?

File: docs/admin/deployment.rst [hunk 5]
```diff
@@ -114,6 +114,9 @@ you create:
    Optional. Set your `Google Cloud Translation API key`_ to use machine translation
    by Google.
 
+``LOCALE_REQUEST_FROM_EMAIL``
+   Optional. Requests about new project locales are sent from this email.
```
Comment (mathjazz) [inline docs/admin/deployment.rst:118]:
```
Optional. Requests for new project locales are sent from this email address.
```

File: docs/admin/deployment.rst [hunk 6]
```diff
@@ -199,6 +202,12 @@ you create:
    Timezone for the dynos that will run the app. Pontoon operates in UTC, so set
    this to ``UTC``.
 
+``VCS_SYNC_NAME``
+  Optional. Default committer's name that Sync uses when a changeset doesn't have authors.
```
Comment (mathjazz) [inline docs/admin/deployment.rst:206]: We actually always use this committer (we only change author). So I'd change this (and similarly below) to:
```
Optional. Default committer's name used when committing translations to version control system.
```

**Comment by mathjazz** [comment]
Something went wrong in that last commit...

**Comment by jotes** [comment]
@mathjazz Yeah, I mistyped a command and pushed the changes too soon. The last commit should be okay.

**Pull request merged**